powerline/config_files/colorschemes/default.json

```diff
@@ -43,6 +43,8 @@
 		"branch_dirty":              { "fg": "brightyellow", "bg": "gray2", "attrs": [] },
 		"branch_clean":              { "fg": "gray9", "bg": "gray2", "attrs": [] },
 		"branch:divider":            { "fg": "gray7", "bg": "gray2", "attrs": [] },
+		"stash":                     { "fg": "brightyellow", "bg": "gray2", "attrs": [] },
```

Comment:
If `stash` is a copy of `branch_dirty` everywhere it is much better to use `"stash": "branch_dirty"` where it is a copy. Same for `stash:divider`, this also avoids changing anything in Vim colorschemes.